documentation/assertions/spy/was-called-on.md

```diff
@@ -40,12 +41,11 @@ expect(obj.spy, 'was always called on', obj);
 
 ```output
 expected spy was always called on { spy: spy }
-  expected invocations( spy(), spy() )
-  to have values satisfying 'was called on', { spy: spy }
+  expected spyCalls to have values satisfying 'was called on', { spy: spy }
 
   invocations(
-    spy(),
-    spy() // expected spy() was called on { spy: spy }
-          //   expected spy to be called with { spy: spy } as this but was called with {}
+    spy() at theFunction (theFileName:xx:yy),
+    spy() at theFunction (theFileName:xx:yy) // expected spy() at theFunction (theFileName:xx:yy) was called on { spy: spy }
+                                             //   expected spy to be called with { spy: spy } as this but was called with {}
   )
```

Comment:
I think it is a good idea to make to satisfy and the general diffs aim for output with width = `preferredWidth - depth * indentationWidth` or use another line if that is impossible. But I would still prefer that trace only to be included in to top-level of the diffs.